projects/js-packages/connection/components/connect-screen/required-plan/visual.jsx

```diff
@@ -81,15 +87,17 @@ const ConnectScreenRequiredPlanVisual = props => {
 						priceBefore={ priceBefore }
 						currencyCode={ pricingCurrencyCode }
 						priceAfter={ priceAfter }
-						infoText={ showConnectButton ? tos : '' }
 					>
 						{ showConnectButton && (
-							<ActionButton
-								label={ buttonLabel }
-								onClick={ handleButtonClick }
-								displayError={ displayButtonError }
-								isLoading={ buttonIsLoading }
-							/>
+							<>
+								<div className="jp-connection__connect-screen-required-plan__toc-info">{ tos }</div>
```

Comment:
should the classname be `jp-connection__connect-screen-required-plan__tos-info`?

Reply:
Indeed it should! I pushed a new commit to address this typo; thank you, @jkguidaven, for noticing this small detail 👍